contracts/Commitment/Commitment.sol

```diff
@@ -612,11 +546,7 @@ contract Commitment is
             ticket.rewardNmk = sub(ticket.rewardNmk, ticketNmk);
             remainingEur = sub(remainingEur, ticketEur);
 
-            // AUDIT[CHF-62] Usage of "+=" instead of "=".
-            //  The totalNmk is always 0 in this place.
-            //  Use `totalNmk = tickerNmk;` or use totalNmk variable instead of
-            //  ticketNmk in this code block.
-            totalNmk = add(totalNmk, ticketNmk);
+            totalNmk += ticketNmk;
```

Comment:
You sill can use `totalNmk = ticketNmk` here. I need confirmation you want to keep the `+=`.